docfx.json

```diff
@@ -400,10 +400,7 @@
                 "docs/framework/**/**.md": "dotnetcontent",
                 "docs/framework/app-domains/**/**.md": "gewarren",
                 "docs/framework/configure-apps/file-schema/network/**/**.md": "ncldev",
-                "docs/framework/configure-apps/file-schema/wcf/**/**.md": "wcfsrvt",
                 "docs/framework/configure-apps/file-schema/winforms/**/**.md": "adegeo",
-                "docs/framework/data/adonet/**/**.md": "dotnetcontent",
```

Comment:
This one is already owned by dotnetcontent (line 400) so no need to break it out here.